Разметка внутренней страницы

biondohod opened the pull request:
---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1896161/tasks/3)

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/1896161/tasks/4)

Commit:
Добавляет разметку главной страницы

Commit:
Merge remote-tracking branch 'origin/master'

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="index.html"><img class="header-navigation-logo" src="" alt="Логотип Седоны"></a>
```
Тут все правильно, молодец, что сделал ссылку в которой логотип, но смотри, мы находимся на главной странице сейчас и плохой практикой является давайть ссылку на страницу, на которой мы находимся - т.е. на саму себя. Поэтому тут все оставляй как есть, только просто удали полностью атрибут href="index.html"

И еще сам логотип не должен быть в списке меню, в меню только ссылки на страницы, а логотип одлжен быть вне этого списка

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+            <a class="navigation-link" href="index.html"><img class="header-navigation-logo" src="" alt="Логотип Седоны"></a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="index.html">Главная</a>
```
По ссылке тоже самое что и выше писал - удалить href

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+            <a class="navigation-link" href="catalog.html">Гостиницы</a>
+          </li>
+        </ul>
+        <ul class="navigation-list navigation-user">
```
у тебя это два разный списка, первый список это список ссылок, а второй это уже пользовательская навигация, а классы для элементов списка у тебя одинаковые, так не пойдет - это внесен сложности в дальнейшем при стилизации

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link button button-brown" href="#">ХОЧУ СЮДА!</a>
```
И вот эта ссылка - она видишь отичается от остальных элементов того же списка, поэтому лучше эту ссылку сделать отдельной от списка

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+    <main>
+      <h1 class="visually-hidden">Седона-город в Аризоне</h1>
+      <img src="" alt="Добро пожаловать в великолепную Седону! Потому что Гранд-Каньон - отстой!">
+      <p class="slogan"><strong>Седона — небольшой городок в Аризоне,<br>
```
Этот элемент лучше сделать как h2

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+      <img src="" alt="Добро пожаловать в великолепную Седону! Потому что Гранд-Каньон - отстой!">
+      <p class="slogan"><strong>Седона — небольшой городок в Аризоне,<br>
+        заслуживающий большего!</strong></p>
+        <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
```
Сюда нужно добавить класс

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+        <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+        <section class="advanteges">
+          <h2 class="visually-hidden">Преимущества</h2>
+          <ul class="advanteges-list">
```
Вот смотри... Тут да все верно получается список преимуществ, их всего 5, но в первом преимуществе есть еще один как бы вложенный список, и он должен быть вложен в первый элемент списка, вот ка кпри примере ниже

```html
<ul>
    <li> // первое преимущество
        <ul>
            <li>
                // первое вложенное преимущество первого преимущества 
            </li>
        </ul>
    </li>
    <li></li> // второе преимущество
    <li></li> // третье преимущество
    <li></li> // четвертое
    <li></li> // пятое
</ul>
```

В остальном все отлично! И еще, нужно убрать везде теги br - это плозая практика. Переносы мы сделаем потом через css

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+        </section>
+        <section class="hotel-search">
+          <h2 class="visually-hidden">Поиск гостиницы</h2>
+          <p><b>Заинтересовались?</b></p>
```
тег b тут тоже лишний. Достаточного одного тега `p` - только ему нужно задать свой класс

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+        </section>
+    </main>
+    <footer class="main-page-footer">
+      <h2 class="footer-title">Подпишитесь на рассылку</h2>
```
Блок "Подпишись на рассылку" нужно сделать как отдельную секцию и она не должна быть в футере, она должны быть в основном контенте в теге `main`

redsmoke19 commented on index.html:
```diff
@@ -5,8 +5,163 @@
+                <span class="visually-hidden">Ютуб</span></a>
+            </li>
+            <li class="footer-contacts-item">
+              <a class="footer-contacts-phone" href="tel:+78128121212">+7 (812) 812-12-12</a>
```
Номер телефона и ссылка на академию это не список - это отдельные элементы

redsmoke19 reviewed (changes requested):
Андрей привет! Ты большой молодец и отлично справился с первой ДЗ. У меня есть несколько замечаний, которые ты можешь прочитать ниже, но в целом все очень и очень хорошо! И еще момент, ты должен правильно форматировать свой код, что бы все теги были на своих местах и что бы была видна вложенноть элементов. Это можно легко сделать - если ты работатешь на Win и в VSCode то просто воспользуйся комбинацией клавиш Alt + Shift + F.

Если будут вопросы по замечаниям - обязательно пиши и все обсудим)

Commit:
Улучшает код(по замечаниям)

Commit:
Добавляет разметку каталога и небольшие правки index.html

(добавление класса тегу <main> и перенос секции с подпиской на рассылку в тег <main>

Commit:
Исправляет неправильное название класса пагинации

redsmoke19 commented on index.html:
```diff
@@ -1,12 +1,165 @@
+          <a class="navigation-link" href="catalog.html">Гостиницы</a>
+        </li>
+      </ul>
+      <ul class="navigation-list navigation-user">
```
Вторые классы тут нужно убрать, не нужно миксовать тут два класса от этого и предыдущего списка, оставь только те что с user

redsmoke19 commented on index.html:
```diff
@@ -1,12 +1,165 @@
+            Если вы осмелитесь,
+            разумеется
+          </p>
+          <img src="" alt="Мост Дьявола">
```
По макету сначала идет картинка, а потом уже текст

redsmoke19 commented on index.html:
```diff
@@ -1,12 +1,165 @@
+        Укажите предполагаемые даты поездки,
+        и мы покажем вам лучшие предложения гостиниц в Седоне
+      </p>
+      <a class="button button-brown" href="#">ПОИСК ГОСТИННИЦЫ В СЕДОНЕ</a>
```
Не нужно в разметке писать заглавными буквами, заглавные приянтно делать через css

redsmoke19 commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+          <a class="navigation-link">Гостиницы</a>
+        </li>
+      </ul>
+      <ul class="navigation-list navigation-user">
```
Со вторыми классами тоже самое что и для главной)

redsmoke19 commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+    </nav>
+  </header>
+  <main class="main-catalog">
+    <header class="catalog-header">
```
Это все таки не херед, а обычная секция, в которой есть заголовок, хлебные крошки, и форма
```html
<section>
    <h1></h1>
    <ul class="breadcrumbs"></ul>
    <form action=""></form>
</section>
```

redsmoke19 commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+  <main class="main-catalog">
+    <header class="catalog-header">
+      <h1 class="catalog-header-title">Гостинницы Седоны</h1>
+      <ul class="breadcrumbs">
```
Вот тут молодец) Правильно назвал этот блок, это действительно хлебные крошки

redsmoke19 commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+          <a class="breadcrumbs-link">Гостинницы</a>
+        </li>
+      </ul>
+      <section class="products-filters">
```
Тут можно без секции обойтись) А сделать секцию, как я писал выше вместо header

redsmoke19 commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+            <ul class="products-filters-list">
+              <li class="products-filters-item">
+                <label class="products-filters-control">
+                  <input class="products-filters-control-input" type="radio" name="type-of-housing" value="hotel "
```
Вся форма у тебя отлично сделана, только вот смотри, у тебя чекбоксы и радио кнопки имеют одинаковый класс, но стилизоваться они будут по разному, поэтому тут для радио кнопок тебе нужно придумать другой класс)

redsmoke19 commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+          </fieldset>
+          <button class="button button-blue" type="submit">Применить</button>
+        </form>
+        <select class="select-control" aria-label="Сортировка">
```
Вот этот селект и ниже список отображения они должны быть уже в секции, так что у тебя ниже, вместе с текстом "Найдено гостиниц" - по макету именно так) Т.е. из этой секции их нужно убрать, а добавить ниже рядом с текстом как по макету, в данном случае это все отдельные блоки которые не имеют отношения к верхней форме и верхней секции

redsmoke19 reviewed (changes requested):
Андрей все круто сделал, и отлично исправил предыдущие замечания. По страницы каталога у меня есть несколько маленьких замечаний, которые нужно исправить, и тогда я все приму и можно будет приступать к следующей теме)
